uber/templates/preregistration/index.html

```diff
@@ -68,6 +68,7 @@
             </ul>
         </td>
         <td>${{ attendee.total_cost }}</td>
+        <td><form action="redeem"><input type="text" name="edit_id" value="{{ attendee.id }}" hidden> <input placeholder="Enter Promo Code" type="text" name="promo"><input type="submit" value="Submit"></form></td>
```

Comment:
That's a really interesting point. On event brite they have a small little button in the corner of the ticket window that says 'enter promotional code' and when you click it the input box appears. Maybe something like that would be more preferred?